src/Components/Server/src/Directory.Build.targets

```diff
@@ -0,0 +1,6 @@
+﻿<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.targets))\Directory.Build.targets" />
+  <!-- Workaround target import for project references to Microsoft.Extensions.FileProviders.Embedded -->
+  <Import
+    Project="$(RepoRoot)src\FileProviders\Embedded\src\build\netstandard2.0\Microsoft.Extensions.FileProviders.Embedded.targets" />
```

Comment:
If this works, :ok: I guess. But my preference would be to do this `<Import />` in the project file rather than adding this additional file

Reply:
I tried that but it doesn't work in the project file since the `PrepareResourceNamesDependsOn` property will be overridden by the SDK.